### PR TITLE
Allow proxying of accounts

### PR DESCRIPTION
**Pull request by atomaka**
ENG-415

Base configuration for proxy accounts in turing.

### CODE REVIEW

File: runtime/neumann/src/lib.rs [hunk 1]
```diff
@@ -361,6 +364,76 @@ impl pallet_identity::Config for Runtime {
...
+	Ord,
+	PartialOrd,
+	Encode,
+	Decode,
+	RuntimeDebug,
+	MaxEncodedLen,
+	scale_info::TypeInfo,
+)]
+pub enum ProxyType {
+	Any = 0,
```
[the remark this answers is not in the document]
Comment (rhuttman) [reply runtime/neumann/src/lib.rs:388]: yes. The default must be the most permissive value: https://github.com/paritytech/substrate/blob/master/frame/proxy/src/lib.rs#L127

**Review by rhuttman — APPROVED** [review]
lgtm assuming those are the correct two types. This also reminds me that we really need a common runtime

**Comment by atomaka** [comment]
> lgtm assuming those are the correct two types. This also reminds me that we really need a common runtime

I actually considered adding a common runtime here, but both Parity and PureStake duplicate this configuration across all of their runtimes instead of leveraging their common runtimes.

**Review by rhuttman — APPROVED** [review]
merge stamp

**Pull request merged**